watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +154,74 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
```

Comment:
This would work, but there is not way to reach it from the `cli`. Given you need to create the `ping` rpcmethod to `watchtower-client/src/main.rs` I'd suggest you move the functionality there. We only place code in `http.rs` if it needs to be used by multiple modules (that is, both main and `retrier`), and I don't think this will be the case, at least not for now.

Reply:
oops! yess i forgot. ok I'll create it inside`main.rs` in plugin.